jest/bowlingScoreCard.test.js

```diff
@@ -0,0 +1,80 @@
+const BowlingScoreCard = require("../lib/bowlingScoreCard");
+
+describe("bowlingScoreCard class", () => {
+  beforeEach(() => {
+    scoreCard = new BowlingScoreCard();
+  });
+
```

Comment:
Good use of beforeEach to auto instantiate

Reply:
Makes the test code cleaner, without having to add the new instance of scoreCard manually for each test. :)